src/Components/AddClass/assets/clock.svg

```diff
@@ -0,0 +1,16 @@
+<svg
```

Comment:
You could consider making your svgs components, that way you could import them as such, too :)